mermithid/processors/TritiumSpectrum/DistortedTritiumSpectrumLikelihoodSampler.py

```diff
@@ -0,0 +1,239 @@
+'''
+Fit Tritium spectrum
+Author: C. Claessens
+Date: Aug 07 2019
+'''
+try:
+    import ROOT
+except ImportError:
+    pass
+
+from morpho.utilities import morphologging, reader
+logger = morphologging.getLogger(__name__)
+
+try:
+    import PhylloxeraPy
+    PhylloxeraPy.loadLibraries(True)
+except ImportError:
+    logger.warning("Cannot import PhylloxeraPy")
+    pass
+
+from morpho.processors.sampling import RooFitInterfaceProcessor
+from mermithid.misc import Constants
+from ROOT import TMath
+
+
+
+class DistortedTritiumSpectrumLikelihoodSampler(RooFitInterfaceProcessor):
```

Comment:
Yu should add some documentation about the processor and the arguments needed for it to run.